smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/range.rs

```diff
@@ -51,7 +51,7 @@ fn move_range_works() {
             &ExecuteMsg::VaultExtension(ExtensionExecuteMsg::ModifyRange(ModifyRangeMsg {
                 lower_price: Decimal::from_str("0.65").unwrap(),
                 upper_price: Decimal::from_str("1.3").unwrap(),
-                max_slippage: Decimal::bps(MAX_SLIPPAGE_HIGH),
+                max_slippage: Decimal::percent(89),
```

Comment:
nitpick: A comment why 89% is appropriate in this test is good imo, since it's not directly a logical value

Reply:
:D something about the test setup. With 90% it failed with a slippage error that missed by one micro-unit